CMake error when trying to open file: /usr/local/bin/laszip/laszip.hpp for reading.

Configuration fails after manually installing LASTools:
``
CMake Error at cmake/modules/FindLASzip.cmake:62 (FILE):
  file Internal CMake error when trying to open file:
  /usr/local/bin/laszip/laszip.hpp for reading.
Call Stack (most recent call first):
  cmake/laszip.cmake:8 (find_package)
  CMakeLists.txt:202 (include)


CMake Error at cmake/modules/FindLASzip.cmake:87 (MESSAGE):
  Failed to open /usr/local/bin/laszip/laszip.hpp file
Call Stack (most recent call first):
  cmake/laszip.cmake:8 (find_package)
  CMakeLists.txt:202 (include)
``

I think this started happening after manually copying LASTools binaries under /usr/local/bin.

--- a/cmake/modules/FindLASzip.cmake
+++ b/cmake/modules/FindLASzip.cmake
@@ -37,8 +37,7 @@ ENDIF()
 
 
 FIND_PATH(LASZIP_INCLUDE_DIR
-  laszip.hpp
-  PATH_PREFIXES laszip
+  laszip/laszip.hpp
   PATHS
   /usr/include
   /usr/local/include

